api/v1alpha1/flavor_types.go

```diff
@@ -77,14 +79,17 @@ type FlavorFilter struct {
 	Name *OpenStackName `json:"name,omitempty"`
 
 	// ram is the memory of the flavor, measured in MB.
+	// +kubebuilder:validation:Minimum=1
```

Comment:
If you haven't already, please can you write a validation to ensure this works as expected? I recall there is some interaction between pointers and validations, but I'm not sure if it applies here as it looks like it has been added to the CRD. I think, e.g. you can't apply MaxLength to `*string`.

Reply:
Done in https://github.com/k-orc/openstack-resource-controller/pull/179